clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -2849,6 +2850,20 @@ public void testUnusedConfigs() {
         }
     }
 
+    @Test
+    public void testUnknownConfigs() {
+        Map<String, Object> props = new HashMap<>();
+        props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9999");
+        props.put(unknownTestConfig, "my_value");
+        ConsumerConfig config = new ConsumerConfig(ConsumerConfig.appendDeserializerToConfig(props, new StringDeserializer(), new StringDeserializer()));
+
+        assertTrue(config.unknown().contains(unknownTestConfig));
```

Comment:
I think we should also verify the `config.unknown()` has only 1 element, that is: `unknownTestConfig`. In current verification, there could be many `unknown` in the configs, but still pass the test. Same as below.

Reply:
I agree with you,thanks.